Testscripts/Linux/dpdkPmdsBasicCheck.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+# This script will run DPDk basic testpmd command in different PMDs and check failures.
+# To run this script constants.sh details must.
+#
+########################################################################################################
+
+HOMEDIR=$(pwd)
+LOGDIR="${HOMEDIR}/DpdkTestPmdLogs"
+CONSTANTS_FILE="./constants.sh"
+UTIL_FILE="./utils.sh"
+DPDK_UTIL_FILE="./dpdkUtils.sh"
+FAILSAFE_FILE="./failsafe.sh"
+
+. ${CONSTANTS_FILE} || {
+	echo "ERROR: unable to source ${CONSTANTS_FILE}!"
+	echo "TestAborted" > state.txt
+	exit 1
+}
+. ${UTIL_FILE} || {
+	echo "ERROR: unable to source ${UTIL_FILE}!"
+	echo "TestAborted" > state.txt
+	exit 2
+}
+. ${DPDK_UTIL_FILE} || {
+	echo "ERROR: unable to source ${DPDK_UTIL_FILE}!"
+	echo "TestAborted" > state.txt
+	exit 2
+}
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+LogMsg "*********INFO: Script execution Started********"
+clientIPs=($(hostname -I))
+if [[ ${clientIPs[1]} == "" ]] || [[ ${clientIPs[2]} == "" ]];
+then
+	LogMsg "Extra NICs doesn't have ips, do dhclient"
```

Comment:
Can you change wording better? I understand what it means but this statement got me thinking-time.

Reply:
Thanks! I've changed it to "The eth1 or eth2 doesn't have IP. Run dhclient to get IP."